src/Marten/Events/AppendEventFunction.cs

```diff
@@ -8,10 +8,12 @@ namespace Marten.Events
     public class AppendEventFunction : Function
     {
         private readonly EventGraph _events;
+        private readonly bool _useAppendEventForUpdateLock;
 
-        public AppendEventFunction(EventGraph events) : base(new DbObjectName(events.DatabaseSchemaName, "mt_append_event"))
+        public AppendEventFunction(EventGraph events, bool useAppendEventForUpdateLock) : base(new DbObjectName(events.DatabaseSchemaName, "mt_append_event"))
```

Comment:
Please change constructor param `useAppendEventForUpdateLock` as an optional param with default value `false`, this way, it becomes binary compatible change.

Reply:
done